src/Request/Request.php

```diff
@@ -56,6 +56,14 @@ class Request implements ConfigAwareInterface, ContainerAwareInterface, LoggerAw
      */
```

Comment:
Can line 54 be updated to mention that you can use a directory now? (Something like "Target file or directory's name")